samples/matter/light_bulb/sample.yaml

```
@@ -28,27 +24,36 @@ tests:
       - nrf5340dk_nrf5340_cpuapp
       - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
-    tags: ci_build
   sample.matter.light_bulb.smp_dfu:
     build_only: true
-    extra_args: CONFIG_CHIP_DFU_OVER_BT_SMP=y
+    extra_args: CONFIG_CHIP_LIB_SHELL=y CONFIG_CHIP_DFU_OVER_BT_SMP=y
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
-    tags: ci_build
-  sample.matter.light_bulb.no_dfu.nrf7002_ek:
-    build_only: true
-    extra_args: SHIELD=nrf7002_ek CONF_FILE=prj_no_dfu.conf
-    integration_platforms:
-      - nrf5340dk_nrf5340_cpuapp
-    platform_allow: nrf5340dk_nrf5340_cpuapp
-    tags: ci_build
   sample.matter.light_bulb.ffs:
     build_only: true
-    extra_args: CONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE=y CONFIG_CHIP_ROTATING_DEVICE_ID=y CONFIG_CHIP_DEVICE_TYPE=257
+    extra_args: CONFIG_CHIP_LIB_SHELL=y CONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE=y CONFIG_CHIP_ROTATING_DEVICE_ID=y CONFIG_CHIP_DEVICE_TYPE=257
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
-    tags: ci_build
+  sample.matter.light_bulb.no_dfu:
+    build_only: true
+    extra_args: CONFIG_CHIP_LIB_SHELL=y CONF_FILE=prj_no_dfu.conf
+    integration_platforms:
+      - nrf21540dk_nrf52840
+    platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
+  # ---------------
+  sample.matter.light_bulb.debug.nrf5340dk:
+    build_only: true
+    extra_args: CONFIG_CHIP_LIB_SHELL=y
+    integration_platforms:
+      - nrf5340dk_nrf5340_cpuapp
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
+  sample.matter.light_bulb.nrf7002_ek:
```

```suggestion
  sample.matter.light_bulb.no_dfu.nrf7002_ek:
```
To keep the convention consistent?

---

OK